File: Day_5/Day_5.java
import java.util.Scanner;

class Day_5 {
    public static void main(String args[]) {
        System.out.print("Enter Integer Value : ");

        Scanner sc=new Scanner(System.in);
        int num=sc.nextInt();

        if(num%2==0)
        {
            System.out.println("even"); 
        }
        else{

            System.out.println("Odd");
        }
       sc.close();
    }
}

File: Day_5/C/Day_5.c
#include<stdio.h>
int main()
{
    int num;
    printf("Enter Integer Number : ");
    scanf("%d",&num);

    if(num%2==0)
    {
        printf("%d",num," is a Even Number.");
    }
    else
    {
        printf("%d ",num," is a Odd Number.");
    }
   
}